modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PistacheServerCodegen.java

```diff
@@ -158,14 +158,21 @@ public CodegenOperation fromOperation(String path, String httpMethod, Operation
         CodegenOperation op = super.fromOperation(path, httpMethod, operation, definitions, swagger);
 
         if (operation.getResponses() != null && !operation.getResponses().isEmpty()) {
-            Response methodResponse = findMethodResponse(operation.getResponses());
+            Map.Entry<String, Response> methodResponse = findMethodResponse(operation.getResponses());
```

Comment:
Since your are changing stuff in many other generators, did you make sure to generate all theses generators? Does it create any changes to the samples?

Reply:
this change affects to other generators but only changes code generated for java play framework, because `successCode` is not used nowhere else